CMakeLists.txt

```diff
@@ -79,13 +86,20 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL Windows)
   set(deployment_target -DDEPLOYMENT_TARGET_WINDOWS)
 endif()
 
+set(foundation_module_name Foundation)
+set(foundation_darwin_link_flags)
+if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  set(foundation_module_name SwiftFoundation)
```

Comment:
Would you want to call this `SwiftFoundation` on Darwin even if there was no ObjC interop?